lib/hash.rb

```diff
@@ -18,9 +20,15 @@ def method_missing(method, *args)
 
     if setter?(method)
       self[prop] = args.first
-    else
-      super(method, args) && return unless key?(prop)
+    elsif key?(prop)
       self[prop]
+    else
```

Comment:
I think we can be a little simpler than this, if we just default to line 24 if `hash_dot_use-default`. What this option is essentially doing is turning all unrecognized methods into keys, so we could just pass those through and let `Hash` handle the implementation.

Reply:
Thanks, good point, I like the revised version better.